### DISCUSSION TITLE
Filer metadata sync with embedded stores

### DISCUSSION THREAD

**Discussion by flomo29** [seaweedfs/seaweedfs#2708] [Q&A]
Hej. When i have multiple filer server with embedded stores, do they syncronize their metadata automatically? I ask, because i want to have multiple filers with different content not overlapping with other filers, so that the blob storage will be split across multiple filers. I want to have one filer with -volumeEncryption for special folders and another filer for the s3 buckets. Do they sync also when i specify different collection names? How to deactivate that behavior?

**Comment by chrislusf** [comment]
If you know Go, you can disable this line https://github.com/chrislusf/seaweedfs/blob/077f831749e6bbf12ff52fd369b493e4dd751817/weed/server/filer_server.go#L164

Later maybe add an option to group filers for different purposes.
